src/Button/Button.stories.js

```diff
@@ -157,25 +137,31 @@ storiesOf("Button", module)
       ],
     };
   })
-  .addWithChapters("Facebook button", () => {
-    const title = text("Title", "Sign in with Facebook");
-    const disabled = boolean("Disabled", false);
-    const loading = boolean("Loading Icon", false);
-    const block = boolean("Block", false);
+  .addWithChapters("Button with icon", () => {
+    const title = text("Title", "Button");
+    const type = select(
+      "Type",
+      {
+        primary: "primary",
+        secondary: "secondary",
+      },
+      "secondary",
+    );
     const size = select(
       "Size",
       {
         small: "small",
         normal: "normal",
         large: "large",
       },
-      "normal",
+      "small",
     );
-    const width = number("Width", 0);
-
+    const showIcon = boolean("Show Icon", true);
+    const source = select("Icon", Object.keys(Icons), "PlusCircle");
+    const onlyIcon = boolean("onlyIcon", false);
```

Comment:
I think, if somebody does not want to show text just doesn't fill `title` prop is this really necessary ?, please make it simple as possible